tests/api_fastapi/core_api/routes/public/test_monitor.py

```diff
@@ -103,3 +103,57 @@ def test_unhealthy_metadatabase_status(self, most_recent_job_mock, test_client):
 
         assert "unhealthy" == body["metadatabase"]["status"]
         assert body["scheduler"]["latest_scheduler_heartbeat"] is None
+
+
+class TestGetHealthDagProcessorEnabled(TestMonitorEndpoint):
```

Comment:
We are testing the same endpoint, I think this should be part of the `TestGetHealth` class.

Reply:
Done